sdk/modules/ui/cpp/include/metavision/sdk/ui/utils/opengl_api.h

```diff
@@ -14,18 +14,29 @@
 
 #ifdef _USE_OPENGL_ES3_
 #include <GLES3/gl3.h>
-// While we keep support for OpenGL, we need to provide a
-// dummy implementation for Glew init function
-#define GLEW_OK 0
-inline int glewInit(void) {
-    return GLEW_OK;
-}
 #else // OpenGL
+#if defined(__APPLE__) && !defined(__linux__)
+#define GL_SILENCE_DEPRECATION
```

Comment:
Not necessarily an action request, but do you remember what is the deprecated thing we are using?